src/utils/compare-events/index.ts

```diff
@@ -0,0 +1,29 @@
+interface PartialEvent {
+  startDate: Date
+  endDate: Date
+}
+
+/**
+ * Compares 2 events. Returns negative if a comes first, positive if b comes first
+ * @param a First Event
+ * @param b Second Event
+ */
+export const compareEvents = (a: PartialEvent, b: PartialEvent) => {
+  const now = new Date()
+
+  const aDiff = Number(now) - Number(a.endDate)
```

Comment:
Can you put the `const now` outside of this method, and change it to `new Date().getTime()`

Then it doesn't have to fetch the date and coerce it to number each time in the loop